bin/busted

```diff
@@ -41,6 +41,13 @@ if not os.getenv("KONG_BUSTED_RESPAWNED") then
     end
   end
 
+  -- create shared dict
+  for line in io.lines("./spec/fixtures/lua_shared_dict") do
+    if #line > 0 then
+      resty_flags = resty_flags .. '--shdict "' .. line .. '" '
```

Comment:
This implementation is not elegant, could we use code like below:
```
dicts = {}
for line in ... do
  table.insert(dicts, '--shdict "' .. line .. '"')
end
resty_flags = resty_flags .. table.concat(dicts, " ")
```

Reply:
done